setup.cfg

```diff
@@ -38,3 +38,18 @@ test =
     pytest-openfiles>=0.5.0
 aws =
     stsci-aws-utils>=0.1.2
+
+[flake8]
+select = F, W, E101, E111, E112, E113, E401, E402, E501, E711, E722
+# We should set max line length to 88 eventually
+max-line-length = 130
+exclude =
+    docs,
+ignore = E203, W503, W504, W605
+
+[tool:pytest]
+minversion = 4.6
+doctest_plus = true
+doctest_rst = true
+text_file_format = rst
+addopts = --show-capture=no --open-files
```

Comment:
You can remove `--show-capture=no` and add a newline.

Should probably remove this option in `jwst` as well.